examples/context-propagation/reactor/src/main/java/example/armeria/contextpropagation/reactor/MainService.java

```diff
@@ -0,0 +1,109 @@
+package example.armeria.contextpropagation.reactor;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.server.HttpService;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Scheduler;
+import reactor.core.scheduler.Schedulers;
+
+public class MainService implements HttpService {
+
+    private final WebClient backendClient;
+
+    public MainService(WebClient backendClient) {
+        this.backendClient = requireNonNull(backendClient, "backendClient");
+    }
+
+    @Override
+    public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) {
+        final Scheduler contextAwareScheduler = Schedulers.fromExecutor(ctx.eventLoop());
+
+        // This logic mimics using a blocking method, which would usually be something like a MySQL
+        // database query using JDBC.
+        final Flux<Long> fetchNumsFromFakeDb =
+                Mono.fromCallable(
+                        () -> {
+                            // The context is mounted in a thread-local, meaning it is available to all
+                            // logic such as tracing.
+                            assert ServiceRequestContext.current() == ctx;
+                            assert !ctx.eventLoop().inEventLoop();
+
+                            try {
+                                // Simulate a blocking API call.
+                                Thread.sleep(50);
+                            } catch (Exception ignored) {
+                                // Do nothing.
+                            }
+                            return Arrays.asList(23L, -23L);
+                        })
+                    // Always run blocking logic on the blocking task executor. By using
+                    // ServiceRequestContext.blockingTaskExecutor, you also ensure the context is mounted
+                    // inside the logic (e.g., your DB call will be traced!).
+                    .subscribeOn(Schedulers.fromExecutor(ctx.blockingTaskExecutor()))
+                    .flatMapIterable(l -> l);
```

Comment:
nit: Could be ignored :-)
```suggestion
                    .flatMapIterable(Function.identity());
```

Reply:
Thanks fixed. 😉